docker/Makefile

```diff
@@ -1,4 +1,4 @@
-VERSION="0.0.1"
+VERSION="1.2.0"
 COMMIT_HASH=`git show -s --format=%H`
```

Comment:
It seems these two variables are no longer needed: VERSION and COMMIT HASH

Reply:
removed